atlas_densities/app/combination.py

```diff
@@ -203,7 +203,7 @@ def combine_markers(annotation_path, hierarchy_path, config):
         ],
         columns=["gene", "volume"],
     )
-    for gene, volume in volumes[["gene", "volume"]]:
+    for gene, volume in volumes[["gene", "volume"]].values:
```

Comment:
the official recommendation is to use `to_numpy()`:
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.values.html